src/documents/NodeAPI.md

```diff
@@ -0,0 +1,91 @@
+---
+title: node.js Support and API
+layout: default
+category: Infusion
+---
+
+# Infusion in node.js
```

Comment:
The Markdown files shouldn't have a level one header explicitly. The text in the "title" attribute above will be used as the level one header for the page in the rendered HTML.